yksgi/src/lib.rs

```diff
@@ -8,19 +8,25 @@ use std::collections::HashMap;
 use std::convert::TryFrom;
 use std::ffi::{c_void, CStr};
 use std::ptr;
+
```

Comment:
Any comments on this line @ltratt? :)

Reply:
I think... let me check... yes, this is extraneous space isn't it?

Reply:
Being serious, we can collapse these into `use std::{convert.., ffi::.., }` and so on rather than having Java-esque multi-line `use`s.

Reply:
I haven't formatted yet. So hopefully the formatter will pick that up. But will double check it does.

Reply:
The formatter probably won't merge them, but I might be wrong. [Let's also run the formatter before raising the PR :)]

Reply:
Yeah, I may have rushed this PR out a bit to quickly, but it was late last night and I wanted to wrap it up.

Reply:
Let's manually merge the `use`s now and be done with it?

Reply:
Fixed in 3724ca6647067bc052cb351907bdff5dd114d2e1.